### ISSUE TITLE
Find and upload Persian News Corpus 

### ISSUE THREAD

**Issue by sehsanm** [sehsanm/embedding-benchmark#1]
*  Find the Persian news Corpus 
*  Define a  corpus file standard. (To be discussed with other Corpus builders) - Most probably one sentence in each line 
*  Upload the  zipped version of the corpus in S3 bucket (Contact @sehsanm  to get the access details) 
*  Write  a code ( code/corpus package) that downloads the  corpus into data/corpus folder and extract the zip file

**Comment by sehsanm** [comment]
Nice tool.  I know that someone  already has collected the Persian news corpus in our NLP lab.  Also note that we should be seeking more than 10 milion sentences. If this tool is able to crawl all of that. Lets build a fresh copy.

**Comment by abb4s** [comment]
the tool has 1,000,000 words  limitation .

**Comment by sehsanm** [comment]
If you are starting to work on this please move it to in progress

**Comment by sehsanm** [comment]
Any progress in this @PoriNiki  ?

**Comment by PoriNiki** [comment]
yes, I sent a pull request for the readme file in the data/corpus folder.
But I didn't change it in the main folder of the repository. I think this would be better not to crowd the main readme file.

**Comment by PoriNiki** [comment]
I'm recently talking with them but, this could take time.

**Comment by sehsanm** [comment]
سلام 
تا انجام شدن کامل این تسک فقط یک 
Readme.md 
فاصله داریم

**Comment by PoriNiki** [comment]
سلام. از طرف من فعلا دیتای مذکور کنسل شد من از این مورد کنار میرم.
